### PR TITLE
Update performance test with ingressgateway.

### PR DESCRIPTION
**Pull request by morvencao**
Update all performance test for istio, replace `istio ingress` and `routerule` with `ingressgateway` and `virtualservice`.
Fix: https://github.com/istio/istio/issues/6623

### CODE REVIEW

**Comment by ldemailly** [comment]
@rkpagadala Any idea why grafana stopped working even once combined?

/lgtm

**Comment by rshriram** [comment]
change branch

**Comment by ldemailly** [comment]
ping

**Comment by ldemailly** [comment]
poke

**Comment by ldemailly** [comment]
it'd be great to get this in so the test works and data can be published for 1.0 onward

File: tools/setup_perf_cluster.sh [hunk 1]
```diff
@@ -33,17 +33,41 @@ function Usage() {
...
+    fi
+}
+
 function List_functions() {
   grep -E "^function [a-z]" "${BASH_SOURCE[0]}" | sed -e 's/function \([a-z_0-9]*\).*/\1/'
 }
 
 if [[ "${BASH_SOURCE[0]}" != "${0}" ]]; then
-  TOOLS_DIR=${TOOLS_DIR:-$(dirname "${BASH_SOURCE[0]}")}
+  TOOLS_ABSPATH=$(abspath ${BASH_SOURCE[0]})
```
Comment (ldemailly) [inline tools/setup_perf_cluster.sh]: what is this for? I sort of remember abspath isn’t everywhere and the red version works on both mac and linux

File: tools/setup_perf_cluster.sh [hunk 2]
```diff
@@ -187,33 +206,6 @@ function get_fortio_k8s_ip() {
   echo "+++ In k8s fortio external ip: http://$FORTIO_K8S_IP:8080/fortio/"
 }
 
-function setup_istio_addons_ingress() {
```
Comment (ldemailly) [inline tools/setup_perf_cluster.sh]: is this working some ofher way now?

Comment (morvencao) [reply tools/setup_perf_cluster.sh]: Seems that `grafana` is still not working

File: tools/setup_perf_cluster.sh (hunk 1 above)
Comment (morvencao) [reply tools/setup_perf_cluster.sh]: `abspath` convert relative path to absolute path to make sure that the scripts and templates can be found.

**Comment by ldemailly** [comment]
any idea why grafana stopped working ?
cc @mandarjog / @douglas-reid 

/lgtm

**Comment by ldemailly** [comment]
restarted the mixer test on circle

**Comment by ayush201** [comment]
Should this fix be put into 1.0.2 release and other prior releases?

**Pull request merged**